README.md

```diff
@@ -11,8 +9,6 @@ This project is been done to test my knowledge on Flexbox and semantic tags used
 - HTML,CSS,FLEXBOX,CSSGRID & JAVASCRIPT
 - Git & Github
 
-## Getting Started
-
 To get a local copy up and running follow these simple example steps.
 
 ### Prerequisites
```

Comment:
- [ ] As the first reviewer requested, you should delete the redundant sections (`Prerequisites`, `Setup`, etc) in your README.
- [ ] _[Optional]_ Also, the text `To get a local copy up and running follow these simple example steps.` belongs to the __`Getting Started`__ section and that header has been deleted.
  - If you want the text to remain, you should add the header and also outline the steps to follow to get a local copy up and running

Reply:
thank you. changes made